### DISCUSSION TITLE
OpenAI-DotNet 7.7.7

### DISCUSSION THREAD

**Discussion by StephenHodgson** [RageAgainstThePixel/OpenAI-DotNet#283] [Announcements]
## What's Changed
* OpenAI-DotNet 7.7.7 by @StephenHodgson in https://github.com/RageAgainstThePixel/OpenAI-DotNet/pull/282
  - Updated static models list
    - Added gpt-4-turbo
    - Marked some models as deprecated since they are no longer available
  - Added temperature to CreateRunRequest and CreateThreadAndRunRequest by @EssentialNRG
  - Fixed temperature to string conversion to be invariant culture for audio requests
  - Fixed type checking built in function tool calls

**Full Changelog**: https://github.com/RageAgainstThePixel/OpenAI-DotNet/compare/v7.7.6...v7.7.7

<hr /><em>This discussion was created from the release <a href='https://github.com/RageAgainstThePixel/OpenAI-DotNet/releases/tag/v7.7.7'>OpenAI-DotNet 7.7.7</a>.</em>

**Comment by Stefan-Bechtel** [comment]
Do you already know when the new functionalities such as file_search and json_response will be implemented?

**Reply by StephenHodgson** [reply]
> json_response 

Implemented

> file_search 

Not implemented until Assistant beta v2 (8.0.0)

**Comment by Stefan-Bechtel** [comment]
Many thanks for the quick reply! 👍
I didn't find anything about json_response in the documentation. Can this be set in the Assistant?
Do you have an approximate estimate of when 8.0.0 will be released?

**Reply by StephenHodgson** [reply]
> Can this be set in the Assistant?

I'll double check, but seems like it should be possible

> Do you have an approximate estimate of when 8.0.0 will be released?

Whenever I can find time to get around to it or a paying client that requires it.
